src/instruments/src/EWD/index.jsx

```diff
@@ -0,0 +1,90 @@
+import ReactDOM from 'react-dom';
```

Comment:
Can keep this file out until it's ready?

Reply:
See below.